### ISSUE TITLE
Need server mechanism for subsample data loads

### ISSUE THREAD

**Issue by duane-edgington** [stoqs/stoqs#145]
Currently, subsample data (eg. nutrient values measured from sampled water, etc) are loaded via csv files. Unfortunately, those files are not served via THREDDS Data Server, since csv files are not directly compatible with THREDDS Data server. Thus, they are currently loaded from a local directory referenced to the location of the loader. eg. 
cl.subsample_csv_base = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'BOG_Data/CANON_OS2015/')
However, this is brittle, mixing data directories with the loader code, and it means checking the data in to the code repository if the loader(s) were to move to a new location. We should either generate CSV netCDF files, and serve those through THREDDS http://www.unidata.ucar.edu/software/thredds/current/tds/TDS.html
as we do with the other data files, or come up with some robust python mechanism to pull the data files from an addressable data store.

**Comment by MBARIMike** [comment]
Yes, serving the data from a web server would be more robust as it would alleviate the need to copy .csv files to be local on the server executing the load. The TDS is a good option as it has an http (fileServer) service that will serve the files directly. We do need to confirm with the owners of the sub-sample processed data files that they are comfortable with their data being available on a web server.

**Comment by MBARIMike** [comment]
Another option is to use the ERDDAP server. It has the advantage of offering multiple export formats and seems to have some traction in the OBIS community. It is being considered for MBON and IOOS/CeNCOOS biological data archiving.

**Comment by MBARIMike** [comment]
Through the MBON project I came across the OBIS-ENV-DATA standard as described in [this workshop report](http://iobis.org/2018/05/18/att/). Seems like something worth exploring. Also, [this commit](https://github.com/MBARIMike/stoqs/commit/db99d5eb8aab06d50e68e08fd06d75a7cf995bcc) mentions the need to improve how *ex situ* data are loaded into STOQS.
